spec/burp_upload_spec.rb

```diff
@@ -1,39 +1,34 @@
 require 'spec_helper'
 require 'ostruct'
 
-describe 'Burp upload plugin' do
-
-  describe Burp::Issue do
-    pending "create some unit tests for the Burp::Issue wrapper class"
-  end
-
-  describe "Importer" do
+module Dradis::Plugins
+  describe 'Burp upload plugin' do
     before(:each) do
       # Stub template service
       templates_dir = File.expand_path('../../templates', __FILE__)
-      expect_any_instance_of(Dradis::Plugins::TemplateService)
+      expect_any_instance_of(TemplateService)
       .to receive(:default_templates_dir).and_return(templates_dir)
 
       # Init services
       plugin = Dradis::Plugins::Burp
 
-      @content_service = Dradis::Plugins::ContentService.new(plugin: plugin)
-      template_service = Dradis::Plugins::TemplateService.new(plugin: plugin)
+      @content_service = Dradis::Plugins::ContentService::Base.new(
```

Comment:
no need to call `Dradis::Plugins` all the time if the spec is in the same module?

Reply:
Updated!